### ISSUE TITLE
Add link option to the Card title in LSX Card Block

### ISSUE THREAD

**Issue by viscosho** [lightspeeddevelopment/lsx-blocks#21]
**Problem**
The card block should have a link option for the Title of the card, this gives more accessibility to the content.

**To Do**
Adding Link Functionality to Title of Card in LSX Card Block

- [ ]  Check if there is a link enabled on the button of the card.
- [ ]  Enable an option to make the title also a link based on this url.
- [ ]  This options should be enabled by default.
- [ ] Make sure this improvement is compatible with the block older version by deprecating code.

**Context**
<img width="392" alt="Screen Shot 2020-02-07 at 9 04 13 PM" src="https://user-images.githubusercontent.com/3129483/74074650-d3efb400-49ed-11ea-916c-16705b1828fa.png">
